README.md

```diff
@@ -748,6 +752,8 @@ make, cmake, gcc, gettext-devel pkg-config, dos2unix, zlib-devel, libexpat1-deve
 
 Download and build libiconv-1.15: [https://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.15.tar.gz](https://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.15.tar.gz)
 
+There is a discussion on the web about installing GTest: [https://github.com/Exiv2/exiv2/issues/575](https://github.com/Exiv2/exiv2/issues/575)
```

Comment:
That was quite intentional.  I mention installing GTest in 3 places:

Section 4.3 Unit Tests
Section 5.3 MinGW
Section 5.4 Cygwin 

I didn't mention it in Section 5.1 Linux because I thought ci/install_dependencies.sh would deal with it.  However I reverted the changes to that and mostly leaving the user to deal with GTest.

Reply:
I see. I am not in favour of having duplication (not even in the documentation). However,  since you took care of most of the documentation changes and it has your personal "touch" I will not complain about that 😉